sop-website/docs/Waveform-Data/waveform-file-format.mdx

```diff
@@ -133,7 +141,7 @@ Stream-based submissions should also have:
 - `start_time`: the (integer, or string) variable that identifies the start of the data stream for each row. Please remember to indicate (in your `SUBMISSION.md`) whether the timestamp corresponds to the start or end of the first sample in the stream.
 - `end_time`: the (integer, or string) variable that identifies the end of the data stream for each row. Please remember to indicate (in your `SUBMISSION.md`) whether the timestamp corresponds to the start or end of the last sample in the stream.
 
-Submit your mapping file(s) to [CHoRUS to WFDB converter repo](https://github.com/chorus-ai/chorus_waveform_conversion). Provide the mapping file(s) under `mappings/<site>/` 
+Submit your mapping file(s) under the `Metadata` folder as depicted above.
```

Comment:
Not sure "depicted" is the right word here, unless there is a diagram somewhere?

Reply:
I was referring to the folder structure diagram towards the top of this page.